src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/ReportFilters/ReportFilters.tsx

```diff
@@ -2,41 +2,90 @@ import { Box, Button } from '@mui/material';
 import { useForm } from 'react-hook-form';
 import { useTranslation } from 'react-i18next';
 
-import { DatePicker, TimePicker } from 'shared/components';
-import { StyledBodyLarge, theme } from 'shared/styles';
+import { DatePicker, TimePicker, DatePickerUiType } from 'shared/components';
+import { StyledBodyLarge, StyledFlexTopCenter, theme, variables } from 'shared/styles';
+import { Switch, TagsInputController } from 'shared/components/FormComponents';
 
-import { StyledForm, StyledTimeText } from './ReportFilters.styles';
+import { StyledTimeText } from './ReportFilters.styles';
+import { FilterValues } from './ReportFilters.types';
 
 export const ReportFilters = () => {
-  const { t } = useTranslation();
-  const { control } = useForm();
-  const commonDatePickerProps = {
-    control,
-    inputSx: { width: '19rem' },
-  };
+  const { t } = useTranslation('app');
+  const { control, setValue, watch } = useForm<FilterValues>({
+    defaultValues: {
+      startDateEndDate: [],
+      moreFiltersVisisble: false,
+      startTime: '',
+      endTime: '',
```

Comment:
don't we have a default value (with empty array) for `identifier` and `versions` ?

Reply:
no

Reply:
why not, let's determine default values

Reply:
it is conditional values